pkg/virtctl/imageupload/imageupload.go

```diff
@@ -346,9 +346,13 @@ func waitUploadPodRunning(client kubernetes.Interface, namespace, name string, i
 }
 
 func createUploadPVC(client kubernetes.Interface, namespace, name, size, storageClass, accessMode string, blockVolume bool) (*v1.PersistentVolumeClaim, error) {
+	if !noCreate && len(size) == 0 {
+		return nil, fmt.Errorf("When create pvc, the size must be specified")
+	}
+
 	quantity, err := resource.ParseQuantity(size)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Validation failed for --pvc-size=%s: %s", size, err)
```

Comment:
Let's change `--pvc-size` to `size`, in case somebody changes the parameter name this might be confusing. Sorry, I suggested `--pvc-size` at the beginning.